new-docs/content/any/project/changelogs/NEXT.md

```diff
@@ -19,6 +19,26 @@ draft: true
   before upgrading.
 {{% /callout %}}
 
+#### Breaking change 1
+
+Previously it was possible to bind predicates to variables, e.g. `p = f(x)`.
+This supported an undocumented feature for unifying predicates,
+e.g. `f(x) = f(1)` would unify `x = 1`.
+
+To avoid confusion with querying rules, binding predicates
+to variables is no longer legal in this release. For example:
+
+```
+f(x,y) if x < 3 and y = g(x);
+```
+
+will now fail at parse time.
+
+A similar thing can be achieved by using lists.
+Instead of `f(x) = f(1)`, you can write `["f", x] = ["f", 1]` if necessary.
+
+#### Breaking change 2
```

Comment:
Trying to come up with good title here but brain failing

Reply:
"Allow querying for unbound variables"?

Reply:
Did behavior *only* change when querying for an unbound?

Reply:
Well, technically the other PR #835 removed the other behaviour this created.

Reply:
Answer after reading #807: Yes, behavior only changed when querying for an unbound, and your proposed changelog entry is 👍